src/WebJobs.Script/Utility.cs

```diff
@@ -520,8 +520,17 @@ private static bool ContainsFunctionWithWorkerRuntime(IEnumerable<FunctionMetada
             return false;
         }
 
-        public static bool CheckAppOffline(string scriptPath)
+        public static bool CheckAppOffline(IEnvironment environment, string scriptPath)
```

Comment:
It would likely make sense to turn this into an environment extension method

Reply:
done.